NoteFind/View Controllers/NewNoteViewController.m

```diff
@@ -109,4 +114,24 @@ - (void)documentPicker:(UIDocumentPickerViewController *)controller didPickDocum
     
 }
 
+- (void)getImageFromCamera:(nonnull UIImage *)img {
+    self.noteImageView.image = img;
+}
+
+- (IBAction)didTapPost:(id)sender {
+    
+    [Note postUserNote:_noteImageView.image withDescription:self.descriptionTextField.text withTitle:self.titleTextField.text withTags:self.tags withBooks:self.books withCompletion:^(BOOL succeeded, NSError * _Nullable error) {
+        if (succeeded) {
+            NSLog(@"note successfully posted!");
```

Comment:
remove NSLog, if this is useful we can create a macro to log only in development